src/plugins/visualizations/public/embeddable/visualize_embeddable.ts

```diff
@@ -393,10 +403,14 @@ export class VisualizeEmbeddable
     }
     this.abortController = new AbortController();
     const abortController = this.abortController;
+
+    const visLayers = await this.fetchVisLayers(expressionParams, abortController);
+
     this.expression = await buildPipeline(this.vis, {
       timefilter: this.timefilter,
       timeRange: this.timeRange,
       abortSignal: this.abortController!.signal,
+      visLayers: visLayers,
```

Comment:
This will fix the test failure, I think.

```suggestion
      visLayers,
```

Reply:
whoops! I had made that change when seeing it fail in the commit hook but forgot to add. Thanks